.bit/tests/test.2.8.js

```diff
@@ -1,11 +1,10 @@
 let uri = undefined
 const fetch = require('node-fetch');
+const functions = require('./functions.js')
 
 uri = process.env.SONGREC_ENDPOINT
 
-if (uri[0] == null) {
```

Comment:
Interestingly this would have never worked

![image](https://user-images.githubusercontent.com/985852/125200198-cc2ae380-e261-11eb-8492-f4da33377ca1.png)

Reply:
Testing for null seems to behave differently on the Actions environment, I'll definitely have to do more testing on that.